shared/protocol/src/main/java/io/pravega/shared/protocol/netty/ByteBufWrapper.java

```diff
@@ -113,14 +120,17 @@ public void copyTo(OutputStream target) throws IOException {
     @Override
     public int copyTo(ByteBuffer byteBuffer) {
         Exceptions.checkNotClosed(this.buf.refCnt() == 0, this);
-        ByteBuf buf = this.buf.duplicate();
+        ByteBuf source = this.buf.duplicate();
```

Comment:
This code is fine, but I don't see how it's any different than the code before. It's not saving any copies.

Reply:
The previous code had a bug in it where it would not update the target buffer's position after the write, thus ending up overwriting the same data over and over again.